src/test/java/nextstep/subway/path/documentation/PathDocumentation.java

```diff
@@ -47,7 +47,7 @@ void path() {
         1250
     );
 
-    when(pathService.findPath(anyLong(), anyLong(), any()))
+    when(pathService.findPath(anyLong(), anyLong(), any(),new LoginMember(1L,"test@test.com","password",7)))
```

Comment:
```suggestion
    when(pathService.findPath(anyLong(), anyLong(), any(PathType.class), nullable(LoginMember.class)))
```

요부분에서 테스트가 깨지고 있는데 이렇게 해보면 어떨까요? 👀